### DISCUSSION TITLE
Welcome to PolarBand2lsl Discussions!

### DISCUSSION THREAD

**Discussion by markspan** [markspan/PolarBand2lsl#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gerken80** [comment]
G'day Mark,
Where did you find the information on what byte array to send the H10 to start the ECG, and what each of the bytes mean?

**Reply by markspan** [reply]
Stolen from:

Pareeknikhil

That's the short answer. He knows best, I just copied the bytearray he used.

**Reply by markspan** [reply]
The link is in the readme.

**Reply by Gerken80** [reply]
Thank you.
